minijinja/src/vm/mod.rs

```diff
@@ -306,7 +318,7 @@ impl<'env> Vm<'env> {
                 Instruction::EmitRaw(val) => {
                     // this only produces a format error, no need to attach
                     // location information.
-                    ok!(out.write_str(val).map_err(Error::from));
+                    ctx_ok!(out.write_str(val).map_err(Error::from));
```

Comment:
This case is intentionally bypassing `ctx_ok!` as you can see from the comment.  This is done for performance reasons.